interactive-repl.md

```diff
@@ -39,11 +39,13 @@ on the toolbar:
  
 ![Escape](./media/RTVS-REPL-escape.png)
 
-While you are executing code in you are building up context (i.e., global
-variables, function definitions, library loads etc.). This context is
-collectively called a Workspace, and you have the ability to load and save
-workspaces. To save a Workspace using a specific filename (the default is
-.RData), click on the Save Workspace button in the REPL:
+## Workspaces and sessions
```

Comment:
We have some confusion now between this and the Workspaces window. Probably should have chosen a different name for the latter, in retrospect - they are two completely different things. But it might need a clarification here, so that people don't get confused. Maybe call this "R workspace"?